src/applications/vaos/appointment-list/components/cards/confirmed/AtlasLocation.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import FacilityAddress from '../../../../components/FacilityAddress';
+
+export default function AtlasLocation({ appointment }) {
+  const vvsAppointment = appointment.legacyVAR.apiData.vvsAppointments[0];
+  const name = `ATLAS facility in ${vvsAppointment.tasInfo.address.city}, ${
```

Comment:
Nit but you could maybe declare `vvsAppointment.tasInfo.address` as a const above since it's repeated a bunch throughout this file.

Reply:
Yep, good point. I'm also going to create a ticket to map the tasInfo into FHIR fields, which should clean some of this up.